dbms/src/main/java/org/frankframework/dbms/MariaDbDbmsSupport.java

```diff
@@ -28,21 +28,16 @@
 public class MariaDbDbmsSupport extends MySqlDbmsSupport {
 
 	private Boolean dbmsHasSkipLockedFunctionality;
-	private String productVersion;
+	private final String productVersion;
 
 	public MariaDbDbmsSupport() {
-		this(false);
+		throw new IllegalStateException("MariaDbDbmsSupport should be instantiated with product-version to determine supported featureset. Calling this constructor is a code-bug.");
```

Comment:
I was wondering how a product version looks like. Should we add an example somewhere? 
Later I saw examples in code below in the PR. But making it more clear, it is wise to add it still.